content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+- @desiredInterval@ := Desired time between updates, in milliseconds. Lowering this value increases the temporal resolution. Location updates whose timestamp differs from the last captured update timestamp by less that this value are to be filtered out. Used to govern the frequency of updates requested from the underlying location provider, as well as the frequency of messages broadcast to subscribers.
+- @minimumDisplacement@ := Minimum positional granularity required, in metres. Lowering this value increases the spatial resolution. Location updates whose position differs from the last known position by a distance smaller than this value are to be filtered out. Used to configure the underlying location provider, as well as to filter the broadcast of updates to subscribers.
+
+h2(#resolution-set). Resolution set
+
+A resolution set, is a set of resolutions that must be provided by the app in order to inform the resolution policy. While developers can define their own resolution sets, Ably also provides a default resolution set that can be used for common use cases.
+
+h3(#default-resolution-set). Default resolution set
+
+The @DefaultResolutionSet@ is the set of resolutions which must be defined in order to satisfy the "default resolution constraints":#default-resolution-constraints. These are required in order to inform the "default resolution policy":#default-resolution-policy.
+
+For the @DefaultResolutionSet@ the following resolutions are expected to be included:
+
+- @farWithoutSubscriber@ := The resolution to select if above the "@proximityThreshold@":#default-resolution-constraints), with no subscribers.
+- @farWithSubscriber@ := The resolution to select if above the "@proximityThreshold@":#default-resolution-constraints), with one or more subscribers.
+- @nearWithoutSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints), with no subscribers.
+- @nearWithSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints), with one or more subscribers.
+
+h2(#resolution-constraints). Resolution constraints
+
+Resolution constraints specify the thresholds and corresponding logical mappings for tracking @Resolution@s that are required by the default @ResolutionPolicy@, which can be adopted by @Publisher@ instances. While developers can define their own resolution constraints, Ably also provides default resolution constraints that can be used for common use cases.
+
+h3(#default-resolution-constraints). Default resolution constraints
+
+The Ably-provided default resolution constraints consist of:
+
+- @DefaultResolutionSet@ := The set of resolutions which must be defined in order to populate the @DefaultResolutionConstraints@.
+- @DefaultProximity@ := This is a distance to destination at which resolution of updates is increased to improve user experience.
```

Comment:
Proximity can be spatial, temporal or both. If the word `distance` means both distance in meters and the time required to arrive at some destination then it's fine (I'm not a native speaker so it isn't that clear to me :wink: )

Reply:
You are quite correct - distance implies distance in metres, that is spatial only. I've hopefully clarified it can be spatial, temporal, or both.